lib/screens/app/wallet/dashboard/wallet_header.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:carousel_slider/carousel_controller.dart';
+import 'package:carousel_slider/carousel_options.dart';
+import 'package:carousel_slider/carousel_slider.dart';
+import 'package:dots_indicator/dots_indicator.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:seeds/constants/app_colors.dart';
+
+import '../../../../widgets/dashboard_widgets/currency_info_card.dart';
+
+class WalletHeader extends StatefulWidget {
```

Comment:
Just add the cosnt keyword when is posible in the constructor to improve performance.

Reply:
Thanks Raul, I will add the const keyword when code is move to V2 and StateLessWidget is being use instead of stateFul